### ISSUE TITLE
on using  'parallel_testrun_count' to run tests in parallel. Even when all my test pass after retry, the build still says test have failed.

### ISSUE THREAD

**Issue by HimanshiTFT** [lyndsey-ferguson/fastlane-plugin-test_center#217]
multi_scan(scheme: 'automationTests', device: 'iPhone 6', derived_data_path: 'build', clean: true, code_coverage: false, try_count: 3, batch_count: 4, parallel_testrun_count: 4)

On adding parallel_testrun_count, failed test before retry also considered for the final result.

**Comment by lyndsey-ferguson** [comment]
Hello @HimanshiTFT, thank you for using the plugin and giving me the opportunity to fix a potential problem.

Can you please update your Issue to answer the questions that appear in new bug reports?  Here is a copy for your convenience:

```
<!-- Thanks for helping _test_center_! Before you submit your issue, please make sure to check the following boxes by putting an x in the [ ] (don't: [x ], [ x], do: [x]) -->

### New Issue Checklist

- [ ] Updated `fastlane-plugin-test_center` to the latest version
- [ ] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [ ] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [ ] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description
<!-- Please include which _test_center_ action you are using. For example, multi_scan, tests_from_junit, etc. -->
<!-- Please include what's happening, expected behavior, and any relevant code samples -->

##### Complete output when running fastlane, including the stack trace and command used
<!-- You can use: `--verbose --capture_output` as the last commandline arguments to get that collected for you -->

<!-- The output of `--verbose --capture_output` could contain sensitive data such as application ids, certificate ids, passwords, authentication tokens, or email addreses, Please make sure you double check the output and replace anything sensitive you don't wish to submit in the issue -->

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>

### Environment

<!-- Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used the `--capture_output` option, please remove this block as it is already included there. -->

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>
```

It would also help if you could provide a sample project that exhibits the behavior? I run tests locally on a sample project of my own and have not anything similar to what is happening in your case.

Thanks!

**Comment by lyndsey-ferguson** [comment]
After waiting for more than 2 weeks for more information on this issue, I am closing this issue. If the above logs and other environmental data can be provided, along with a sample project/Fastfile, I can re-open this Issue.
